src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -146,20 +146,50 @@ partial class BoundForEachStatement : IForEachLoopStatement
 
     partial class BoundSwitchStatement : ISwitchStatement
     {
+        private static readonly ConditionalWeakTable<BoundSwitchStatement, object> s_switchSectionsMappings =
```

Comment:
Yes.  Things are now strongly typed.  Unlike today where you are just using 'object' and having to put in explicit casts to work around that.